src/windows_registry.rs

```diff
@@ -810,10 +816,12 @@ mod impl_ {
             "16.0" => {
                 find_msbuild_vs16("x86_64-pc-windows-msvc").is_some()
                     || find_msbuild_vs16("i686-pc-windows-msvc").is_some()
+                    || find_msbuild_vs16("aarch64-pc-windows-msvc").is_some()
             }
             "15.0" => {
                 find_msbuild_vs15("x86_64-pc-windows-msvc").is_some()
                     || find_msbuild_vs15("i686-pc-windows-msvc").is_some()
+                    || find_msbuild_vs15("aarch64-pc-windows-msvc").is_some()
```

Comment:
I'm not sure how common is it to use VS2017 to build ARM64 code, but to my knowledge the latest version of it (15.9) supports it, and vswhere.exe should determine it. So I guess it's best to probe for it in VS2017 as well, what do you think?